ocaml/xapi/xapi_cmd_result.ml

```diff
@@ -36,4 +36,9 @@ let find key results = RMap.find_opt key results
 let of_output_opt ~sep ~key ~lines =
   String.split_on_char '\n' lines
   |> List.fold_left (fun results line -> add ~sep line results) empty
-  |> find key
+  |> RMap.find_opt key
+
+let of_output ~sep ~key lines =
```

Comment:
Why do we need this when we have ` of_output_opt` already? Can we not just call that function and use the result?

Reply:
Lots of other structures like `Map` provides both, 
I think if we provides both, the caller can decides which one to use, handle the return value or exception.

Reply:
Sure, but in that case you can implement `of_output` by using `of_output_opt`, you can avoid copy/pasting the code.